apis/oss/oss.go

```diff
@@ -0,0 +1,19 @@
+/*
+
```

Comment:
Looks like these license headers may be spaced differently than existing / generated files.

Reply:
Yes, I generated the copyright in Goland https://www.jetbrains.com/help/idea/copyright.html.
Are there any convenient way to generate the one exactly same as the existing ones? Thanks.